docs/standard/numerics.md

```diff
@@ -14,63 +14,68 @@ ms.assetid: dfebc18e-acde-4510-9fa7-9a0f4aa3bd11
+|<xref:System.SByte?displayProperty=nameWithType>|Signed|1|-128|127|
+|<xref:System.UInt16?displayProperty=nameWithType>|Unsigned|2|0|65,535|
+|<xref:System.UInt32?displayProperty=nameWithType>|Unsigned|4|0|4,294,967,295|
+|<xref:System.UInt64?displayProperty=nameWithType>|Unsigned|8|0|18,446,744,073,709,551,615|
+
 Each integer type supports a set of standard arithmetic operators. The <xref:System.Math?displayProperty=nameWithType> class provides methods for a broader set of mathematical functions.
 
-You can also work with the individual bits in an integer value by using the <xref:System.BitConverter?displayProperty=nameWithType> class.  
+You can also work with the individual bits in an integer value by using the <xref:System.BitConverter?displayProperty=nameWithType> class.
 
-> [!NOTE]  
+> [!NOTE]
 > The unsigned integer types are not CLS-compliant. For more information, see [Language Independence and Language-Independent Components](language-independence-and-language-independent-components.md).
 
 ## BigInteger
 
 The <xref:System.Numerics.BigInteger?displayProperty=nameWithType> structure is an immutable type that represents an arbitrarily large integer whose value in theory has no upper or lower bounds. The methods of the <xref:System.Numerics.BigInteger> type closely parallel those of the other integral types.
-  
+
 ## Floating-point types
 
-.NET includes three primitive floating-point types, which are listed in the following table:
-  
-|Type|Size (in bytes)|Approximate range|Precision|  
-|----------|--------|---------------------|--------------------|  
-|<xref:System.Single?displayProperty=nameWithType>|4|±1.5 x 10<sup>−45</sup> to ±3.4 x 10<sup>38</sup>|~6-9 digits|  
-|<xref:System.Double?displayProperty=nameWithType>|8|±5.0 × 10<sup>−324</sup> to ±1.7 × 10<sup>308</sup>|~15-17 digits|  
-|<xref:System.Decimal?displayProperty=nameWithType>|16|±1.0 x 10<sup>-28</sup> to ±7.9228 x 10<sup>28</sup>|28-29 digits|  
-  
-Both <xref:System.Single> and <xref:System.Double> types support special values that represent not-a-number and infinity. For example, the <xref:System.Double> type provides the following values: <xref:System.Double.NaN?displayProperty=nameWithType>, <xref:System.Double.NegativeInfinity?displayProperty=nameWithType>, and <xref:System.Double.PositiveInfinity?displayProperty=nameWithType>. You use the <xref:System.Double.IsNaN%2A?displayProperty=nameWithType>, <xref:System.Double.IsInfinity%2A?displayProperty=nameWithType>, <xref:System.Double.IsPositiveInfinity%2A?displayProperty=nameWithType>, and <xref:System.Double.IsNegativeInfinity%2A?displayProperty=nameWithType> methods to test for these special values.
+.NET includes the following primitive floating-point types:
+
+|Type|Size (in bytes)|Approximate range|Precision|
+|----------|--------|---------------------|--------------------|
+|<xref:System.Half?displayProperty=nameWithType>|2|||
```

Comment:
@pgovind Can you help me add the approximate range and precision values?

Reply:
I was going to suggest add the appropriate range too:

- [min](https://review.docs.microsoft.com/en-us/dotnet/api/system.half.minvalue?view=net-5.0&branch=main#System_Half_MinValue)
- [max](https://review.docs.microsoft.com/en-us/dotnet/api/system.half.maxvalue?view=net-5.0&branch=main#System_Half_MaxValue)

Reply:
Called out elsewhere already. Just for complete-ness: The range is `+/-65504`

Reply:
Thanks. For System.Single, this table shows the low end of the range as 1.5x10<sup>-45</sup>. But the [API ref docs show the MinValue](https://docs.microsoft.com/en-us/dotnet/api/system.single.minvalue?view=net-5.0) as -3.40282347E+38. Which one is correct?

Reply:
`MinValue` is `-3.40282347E+38`. However, there is some nuance here about normal and subnormal numbers. @tannergooding, we should decide what we want to docs to state the range as (applies to Double too)

Reply:
One is giving the minimum value and the other is giving the smallest representable magnitude (that is the smallest +/- value which is not zero)

Reply:
Indeed, but the way the range is specified in the docs right now is misleading IMO. Does it make more sense here to just state the range as `+-3.40282347E+38`. The documentation for `Epsilon` at https://docs.microsoft.com/en-us/dotnet/api/system.single.epsilon?view=net-5.0 is quite good already.

Reply:
> Indeed, but the way the range is specified in the docs right now is misleading IMO

Agreed, it can be confusing depending on how you read it and how familiar you are with IEEE 754 floating-point.

Reply:
Cool. I think we should just state the range here as `+-3.4E+38` for Single, `+-1.7E+308` for Double and `+-7.9E+28` for Decimal.